draft-ietf-ppm-dap.md

```diff
@@ -1554,12 +1562,17 @@ struct {
 where `prepare_resps` are the outbound prep steps computed in the previous step.
 The order MUST match `AggregationJobInitReq.prepare_inits`.
 
-The Helper responds to the Leader with HTTP status code 201 Created and a body
-consisting of the `AggregationJobResp`, with media type
-"application/dap-aggregation-job-resp".
+After receiving the response to its `AggregationJobInitReq`, the Leader makes
+an HTTP GET request to the aggregation job URI to check on the status of the
+aggregation job. If the aggregation job is ready, the Helper responds with
+HTTP status 200 OK and a body consisting of an `AggregationJobResp`, with media
+type "application/dap-aggregation-job-resp". If the aggregation job is not
+finished yet, the Helper responds with HTTP status 202 Accepted. The response
+MAY include a Retry-After header field to suggest a polling interval to the
+Leader.
 
 Changing an aggregation job's parameters is illegal, so further requests to
-`PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id}` for the same
+`PUT /tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` for the same
 `aggregation-job-id` but with a different `AggregationJobInitReq` in the body
 MUST fail with an HTTP client error status code.
```

Comment:
@cjpatton I believe this paragraph answers your question "How does the Helper multiple PUTs for the same aggregation job?"

PUT multiple times is legal for idempotence, but mutation is illegal.